deadd-notification-center.cabal

```diff
@@ -33,14 +37,15 @@ library
                      , NotificationCenter.Glade
                      , TransparentWindow
                      , Helpers
+  other-modules:       Paths_deadd_notification_center
+  autogen-modules:     Paths_deadd_notification_center
   build-depends:       base >= 4.7 && < 5
                      , regex-tdfa
-                     , gtk3 >= 0.15.4
                      , transformers
-                     , cairo
                      , filepath
                      , haskell-gi
                      , haskell-gi-base
+                     , haskell-gettext
                      , gi-cairo
                      , gi-pango
                      , gi-glib >= 2.0.17
```

Comment:
Sorry, can not commit on line 53 due to github... but you prob have to add >= 2.0.28 for gi-gdkpixbuf to resolve the compilation issue

Reply:
I added gi-gdkpixbuf >= 2.0.26 as Nix builds fine using gi-gdkpixbuf-2.0.26. Hopefully this is fine by you.

Reasoning:
According to the issue you linked it, the compilation issue occurs based on the installed introspection data. That explains why Stack also built fine on my machine but failed on the CI. The installed introspection data on the CI is incorrect. I pinned the gi-gdkpixbuf to 2.0.28 for Stack to work around the CI issue.